src/App.jsx

```diff
@@ -14,33 +15,77 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_BY_ALPHABET = 'alphabet';
+const SORT_BY_LENGTH = 'length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+export const App = () => {
+  const [sortfiled, setSortfiled] = useState('');
```

Comment:
There is a typo in the variable name `sortfiled`. It should be `sortField` for better readability and to avoid confusion.